src/nidcpower/metadata/functions_addon.py

```diff
@@ -70,6 +78,8 @@
     'revision_query':               { 'parameters': { 1: { 'size': {'mechanism':'fixed', 'value':256}, }, 
                                                       2: { 'size': {'mechanism':'fixed', 'value':256}, }, }, }, # From documentation
     'GetChannelName':               { 'parameters': { 3: { 'size': {'mechanism':'ivi-dance', 'value':'bufferSize'}, }, }, },
+    'SetSequence':                  { 'parameters': { 1: { 'size': {'mechanism':'passed-in', 'value':'Size'}, }, }, },
```

Comment:
The problem above is probably that you misspelled this parameter.
The code-generator should die if mechanism is "passed-in" _and_ value is not found in the parameter list. If you confirm this was the issue, we can look into adding that check.

Reply:
This was correctly spelled, but will be resolved by the len feature @texasaggie97 is working on.